packages/components/stories/List.js

```diff
@@ -393,6 +393,21 @@ storiesOf('List', module)
 			</div>
 		);
 	})
+	.add('Virtualized - list with progress', () => {
+		const loadingListProps = cloneDeep(props);
+		loadingListProps.list.inProgress = true;
+		return (
+			<div style={{ height: '60vh' }}>
+				<h1>List</h1>
+				<p>When the list is loading, a CircularProgress is displayed instead of the rows.</p>
+				<IconsProvider defaultIcons={icons} />
+				<h2>Table</h2>
+				<List {...loadingListProps} virtualized />
+				<h2>Large</h2>
+				<List {...loadingListProps} displayMode="large" virtualized />
```

Comment:
I think there is a problem in your example:
Table -> virtualized 
Large -> same without virtualized ?
I think just one is enough (virtualized will becomes the default soon)

Reply:
@jmfrancois  The second one is large with virtualized
But agree, you can remove the large example